apps/frontend/src/pages/collection/[id].vue

```
@@ -623,6 +567,21 @@ async function unfollowProject(project) {
   projects.value = projects.value.filter((x) => x.id !== project.id);
 }
 
+function makeEdits(event) {
+  try {
+    icon.value = event.icon_data;
+    deletedIcon.value = event.icon_removed;
+
+    name.value = event.name;
+    summary.value = event.description;
+    visibility.value = event.visibility;
+
+    saveChanges();
+  } catch {
+    console.error("uhh.... This isn't good.");
```

This should throw the error into a notification pop-up.

---

Latest commit should resolve this.